consul/test_consul.py

```diff
@@ -366,6 +367,44 @@ def mock_get_coord_nodes(self, instance):
             }
         }]
 
+    def mock_get_health_check(self, instance, endpoint):
+        return [{
+            "ModifyIndex": 75214492,
+            "CreateIndex": 75214492,
+            "Node": "node-1",
+            "CheckID": "server-loadbalancer",
+            "Name": "Service 'server-loadbalancer' check",
+            "Status": "critical",
+            "Notes": "",
+            "Output": "CheckHttp CRITICAL: Request error: Connection refused - connect(2) for \"localhost\" port 80\n",
+            "ServiceID": "server-loadbalancer",
+            "ServiceName": "server-loadbalancer"
+        },
```

Comment:
since we have conditional branches evaluating `ServiceName` and `ServiceID` might be worth it adding a mock payload with those fields empty